crates/stdarch-gen/neon.spec

```diff
@@ -2429,8 +2594,8 @@ aarch64 = pmull
 generate poly64x2_t:poly64x2_t:p128
 
 /// Vector long multiply with scalar
-name = vmull
-n-suffix
+name = vmull_n
+no-q
```

Comment:
Maybe this change should be documented in the PR or in an extra commit as they change the names of public intrinsics (to their correct names).

Reply:
Thanks for pointing it out! I added relevant documents to the description of the PR.